src/twisted/newsfragments/9778.bugfix

```diff
@@ -0,0 +1 @@
+twisted.internet.process.BaseProcess will be removed from twisted.internet.process.reapProcessHandlers before calling ProcessProtocol.processEnded()
```

Comment:
I don't think that this will make sense to most users &mdash; `BaseProcess` is really an implementation detail (I'd guess it would be private if this code had been written recently).

Perhaps:

> On POSIX systems a reaped process's transport is removed from twisted.internet.process.reapProcessHandlers before the associated twisted.internet.interfaces.IProcessProtocol is notified.

I am tempted to say it should be a `.misc`, even. It's basically a cleanup.